apps/aecore/src/aec_chain_state.erl

```diff
@@ -875,6 +875,7 @@ grant_fees(Node, Trees, Delay, FraudStatus, State) ->
     KeyFees  = db_get_fees(hash(KeyNode2)),
     Beneficiary1 = node_beneficiary(KeyNode1),
     Beneficiary2 = node_beneficiary(KeyNode2),
+    BeneficiaryProtocolFoundation = aec_governance:protocol_beneficiary(),
```

Comment:
Should we make this a list? It might be that the community decides on multiple "extra" beneficiaries.

Reply:
OK, good idea.

Reply:
@hanssv however... now I think it raises follow up issues: what's the ration of split between beneficiaries? We will check how much time we have to deliver it and act accordingly.

Reply:
A list of beneficiaries and ratio?

Reply:
yes, actually this is how it is done in stratum